### PR TITLE
News article

### PR DESCRIPTION
**Pull request by tmampa**
In this project, I completed the following:

Build a clone of the original New York Times article page.
Embed videos using HTML 5 tag.
Position elements using Flexbox, Grid, Float, and Position Absolute/Relative/Fixed.
Deploy on GitHub pages.

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @tmampa  :wave:

__You've really done well so far. Here're the major highlights of this PR:__

- All your linter checks are running and passing successfully :100: 
- You have a descriptive PR title and summary :ok_hand: 
- Your Readme looks great! :+1:
- Your webpage contents are well structured :+1: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] The `display: flex` rule occurs in about eight(8) different parts of your CSS blocks. You can avoid this repetition by declaring a block (i.e. .flex { display: flex }) and apply the flex class to those elements you want to display as flex. Always keep your code DRY
- [ ] Kindly remove the bullets for all list items in the bottom navigation section. You can check the original page and ensure to implement similar behavior on your page.
- [ ] The `Suggested newsletters for you`, `Most popular` and `More in Space and Astronomy` sections are completely missing from your page. Please add them.
conflict

- [ ] The `Editor`s Picks` section is also missing. Kindly add it. If you cannot find this section using firefox, please use a different browser (e.g. chrome) and you should see it.
conflict

- [ ] Do not forget to use <a href="https://www.w3schools.com/tags/tag_aside.asp" rel="nofollow noreferrer noopener" target="_blank"><code>&lt;aside&gt;</code> tag</a> for <code>Editor's picks</code> section on the righmost side. (<a href="https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png">screenshot</a> from <a href="https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/" rel="nofollow noreferrer noopener" target="_blank">Pathwright</a>)
- [ ] Check my inline comments below for other changes.

**[ OPTIONAL CHANGES ]**

- [ ] You should be careful with overusing the `id` attribute as a CSS selector. If your page is well cascaded and class selectors are used appropriately, I believe you do not have to use that `id` attribute as a CSS selector. This is because `id` selectors have higher precedence than class selectors and if care is not taken, it can mess up your styling in large projects. Check [this article](https://speckyboy.com/good-bad-css-practices/) for other CSS bad practices.

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project.**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
+vscode
+test.md
+package-lock.json
```
Comment (descholar-ceo) [inline .gitignore:4]: [required] Remove this `package-lock.json` file from the gitignore file

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+node_modules
+vscode
```
Comment (descholar-ceo) [inline .gitignore:2]: [required] it should be `.vscode` instead `vscode` => notice the dot at the beginning

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (descholar-ceo) [inline .vscode/settings.json:3]: [required] You should remove this `.vscode` folder from your repo. To achieve that you can run `git rm -rf .vscode` and add `.vscode` in gitignore and then commit the changes

File: index.html [hunk 4]
```diff
@@ -0,0 +1,653 @@
...
+                                  Jeff J Mitchell/Getty Images
+                              </div>
+                          </div>
+                      </div>
+                      <div class="image-title">
+                          NASA's Last Rocket
+                          <div>March 24</div>
+                      </div>
+                  </div>
+              </div>
```
Comment (descholar-ceo) [inline index.html:494]: [required] the boxes in this section, I mean image and text should be links, this means they use anchor tags. Check carefully the original page

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @tmampa  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone almost looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] There is a long white space between the navigation bar and the main content of the page, kindly remove that space and make sure that the right side editor's pick is floating on the rightmost part of the view port

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-03-26 14-23-31 Screenshot from 2021-03-26 14-23-51 | Screenshot from 2021-03-26 14-26-07 |

:point_right: Could you see the difference :question: :question: :question: 
___

### :warning: Warning: According to the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#) you have 1 code review left for this project, make sure that you implement all of the required changes carefully and request another review while you are sure that it's ready. :warning: 

_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

File: .gitignore (hunk 1 above)
Comment (tmampa) [reply .gitignore]: @descholar-ceo I fixed all the requested changes, however I cannot see the white space between my nav and the rest of the page, could you please check for me if you can see it and if I'm ready to submit another review

**Comment by tmampa** [comment]
@descholar-ceo  fixed all the requested changes, however I cannot see the white space between my nav and the rest of the page, could you please check for me if you can see it and if I'm ready to submit another review
Screenshot from 2021-03-26 16-03-25

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @tmampa :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**